stdlib/http/src/main/ballerina/src/http/websocket/websocket_connector.bal

```diff
@@ -43,51 +45,61 @@ type WebSocketConnector object {
         return externPushText(self, java:fromString(text), finalFrame);
     }
 
-    # Push binary data to the connection.
+    # Pushes binary data to the connection.
     #
     # + data - Binary data to be sent
     # + finalFrame - Set to `true` if this is a final frame of a (long) message
-    # + return - `error` if an error occurs when sending
+    # + return - An `error`. If an error occurs while sending the binary message to the server, that message
+    #            will be lost
     public function pushBinary(byte[] data, boolean finalFrame) returns WebSocketError? {
         return externPushBinary(self, data, finalFrame);
     }
 
-    # Ping the connection.
+    # Pings the connection.
     #
-    # + data - Binary data to be sent.
-    # + return - `error` if an error occurs when sending
+    # + data - Binary data to be sent
+    # + return - An `error`. If an error occurs while sending the ping frame to the server, that frame will be lost
```

Comment:
Can we be consistent when documenting the return value? Here we have a full stop after the `error` and starts a  new sentence with the word `If`.  But in some places (for ex: Line 92) same sentence has continued.